### DISCUSSION TITLE
How to use multi-factor authentication and remember device for example 30 days

### DISCUSSION THREAD

**Discussion by vmeganathan81** [MicrosoftEdge/WebView2Feedback#3645] [General]
How to use multi-factor authentication and remember device for example 30 days, do we have code snippet for this ?

**Comment by vmeganathan81** [comment]
@yildirimcagri any update on this?

**Comment by ghost** [comment]
Hi @vmeganathan81 , since WebView2 saves cookies similar to browser, MFA and device remembering should work by default for pages without need for any configuration specific to WebView2.
